filters/openpolicyagent/evaluation.go

```diff
@@ -0,0 +1,76 @@
+package openpolicyagent
+
+import (
+	"context"
+	"time"
+
+	ext_authz_v3 "github.com/envoyproxy/go-control-plane/envoy/service/auth/v3"
+	"github.com/open-policy-agent/opa-envoy-plugin/envoyauth"
+	"github.com/open-policy-agent/opa-envoy-plugin/opa/decisionlog"
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/server"
+	"github.com/open-policy-agent/opa/tracing"
+	"github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
+)
+
+func (opa *OpenPolicyAgentInstance) Eval(ctx context.Context, req *ext_authz_v3.CheckRequest) (*envoyauth.EvalResult, error) {
+	var err error
```

Comment:
this you don't need anymore, because you use `:=` 2 lines below to have `err` defined